packages/mobile/src/store/storeContext.ts

```diff
@@ -5,12 +5,14 @@ import { apiClient } from 'app/services/audius-api-client'
 import { audiusBackendInstance } from 'app/services/audius-backend-instance'
 import { fingerprintClient } from 'app/services/fingerprint'
 import { remoteConfigInstance } from 'app/services/remote-config'
+import { walletClient } from 'app/services/wallet-client'
 
 export const storeContext: CommonStoreContext = {
   getLocalStorageItem: async (key) => AsyncStorage.getItem(key),
   setLocalStorageItem: async (key, value) => AsyncStorage.setItem(key, value),
   remoteConfigInstance,
   audiusBackendInstance,
   apiClient,
```

Comment:
just curious, why are the first two suffixed with Instance and the rest isn't?

Reply:
We started following that pattern with remoteConfigInstance, but I would be down to remove `Instance` or add it to the other ones. Either way

Reply:
yeah its a good call, im just going off existing naming conventions. in the future would be down to rename all this to something more consistent, do you have thoughts? maybe remove the "instance" since that's just a lot of extra chars?

Reply:
oh clutch ahah we were typing the sameee thing <3